frontend/public/components/factory/list-page.jsx

```diff
@@ -68,15 +72,17 @@ export class ListPageWrapper_ extends React.PureComponent {
       rowFilters,
     } = this.props;
     const data = flatten ? flatten(this.props.resources) : [];
+    const filteredData = this.getFilteredRows(this.props.filters.name, data);
+    const intersection = _.intersection(data, filteredData);
```

Comment:
I think this is unnecessary. Won't the intersection of `data` and `filteredData` always be the same as `filteredData`?